crates/wasm/fs/private/forest.rs

```diff
@@ -23,4 +35,48 @@ impl PrivateForest {
     pub fn new() -> PrivateForest {
         Self(Rc::new(WnfsPrivateForest::default()))
     }
+
+    #[wasm_bindgen]
+    pub fn get(
+        &self,
+        saturated_namefilter_hash: Uint8Array,
+        revision_key: Uint8Array,
```

Comment:
`wasm-bindgen` allows us to pass `Vec<u8>`.

```suggestion
        saturated_namefilter_hash: Vec<u8>,
        revision_key: Vec<u8>,
```